internal/test/wsu/wsu_test.go

```diff
@@ -328,13 +338,11 @@ func testCNIConfig(t *testing.T, node *v1.Node, vm e2ef.WindowsVM, ansibleTempDi
 	assert.Contains(t, cniConfigFileContents, requiredServiceNetwork, "CNI config does not contain service network")
 }
 
-// testFilesCopied tests that the files we attempted to copy to the Windows host, exist on the Windows host
-func testFilesCopied(t *testing.T, vm e2ef.WindowsVM, ansibleTempDir string) {
-	expectedFileList := []string{"kubelet.exe", "worker.ign", "wmcb.exe", "hybrid-overlay.exe", "kube.tar.gz"}
-
+// testFilesExist tests that the files we expect, exist on the Windows host
+func testFilesExist(t *testing.T, vm e2ef.WindowsVM, expectedFileList []string, directoryPath string) {
```

Comment:
Given you are testing if remote files exists, how about calling this `testRemoteFilesExist`?